service/src/main/java/bio/terra/workspace/db/WorkspaceDao.java

```diff
@@ -45,7 +46,7 @@
 public class WorkspaceDao {
   /** SQL query for reading a workspace */
   private static final String WORKSPACE_SELECT_SQL =
-      "SELECT workspace_id, user_facing_id, display_name, description, spend_profile, properties, workspace_stage"
+      "SELECT workspace_id, user_facing_id, display_name, description, spend_profile, properties, workspace_stage, workspace_lineage"
```

Comment:
nit: you can use the new triple quote format to make this prettier. Like:
```
  """
  SELECT workspace_id, user_facing_id, display_name, description, spend_profile, properties, 
    workspace_stage, workspace_lineage
  FROM workspace
  """;
```
I have been gradually switching over the SQL when I am in the neighborhood..